### PR TITLE
Add Largest Contentful Paint documentation for Lighthouse

### LINKED ISSUE #1628: content: Lighthouse doc for Largest Contentful Paint
**Author**: @kaycebasques 
**Reviewer**: @mfriesenhahn  

Note that this is different than #1570.

**Target publish date:** 2019-10-17 (Chrome 79 Branch Point)

- [x] Check this box if this is a hard deadline.
- [x] Check this box if you'd like your reviewer to fix grammar and punctuation issues for you.

**Process**

Check out the [web.dev writing process](https://web.dev/handbook/quick-start/).

When you're ready, make a copy of [this template](https://docs.google.com/document/d/1lgaNIEnXZf-RB8_p9RK22QEgpXJqnu77pLWVWVy4nuw/edit)
and fill in the proposal and outline sections. When you're finished, drop a link
to the doc in the 1st step below and check the box. A member of the web.dev team
will assign this ticket to themselves and work as your reviewer to help you
complete the remaining steps.

- [x] 1. I've done my proposal <add a link to your proposal doc>
- [x] 2. Proposal approved — _reviewer checks this_
- [x] 3. I've added a draft to my proposal doc
- [x] 4. Draft approved — _reviewer checks this_
- [x] 5. I've submitted a pull request #1639 
- [ ] 6. Pull request approved — _reviewer checks this_

### PR DESCRIPTION
**Pull request by kaycebasques**
Fixes #1628 

Changes proposed in this pull request:

- Adds `/lighthouse-largest-contentful-paint/`
- Updates `/lighthouse-performance/` to link to the new doc

### CODE REVIEW

**Comment by kaycebasques** [comment]
# Rationale for why this doc should exist even though it's short

The [LCP](https://web.dev/largest-contentful-paint) doc by Phil Walton that already exists on web.dev is excellent. One might wonder why we need this doc. In general, all of the Lighthouse documentation have the same structure, which we believe results in a more consistent UX. Phil's post also doesn't cover any Lighthouse specific stuff, like how exactly Lighthouse calculates LCP data. So rather than hacking Phil's doc to cover an audience that it wasn't intended for, our general approach is to just create a minimal Lighthouse doc, and then link out to Phil's great content where relevant.

File: src/site/content/en/lighthouse-performance/lighthouse-largest-contentful-paint/index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+need to calculate LCP. It just uses the [Largest Contentful Paint API][api] to get LCP
+data from Chrome.
+
+<!-- TODO explain how the LCP data is mapped to a score between 0 and 100. -->
+
+{% include 'content/lighthouse-performance/scoring.njk' %}
+
+## How to improve your LCP score
+
+See [How to improve Largest Contentful Paint on your site][improve].
```
Comment (kaycebasques) [inline src/site/content/en/lighthouse-performance/lighthouse-largest-contentful-paint/index.md]: See [this comment](https://github.com/GoogleChrome/web.dev/pull/1639#issuecomment-540241698) if you're wondering why this doc should exist even though it's minimal and mostly just links out to web.dev/largest-contentful-paint/

**Comment by kaycebasques** [comment]
Hi @connorjclark since you wrote the first PR for LCP I'm guessing that you're the best PoC for this new LCP content.

Please provide a technical review on this PR:

- [ ] Reply to the comments where I ask you for technical details
- [ ] Check that the explanation of what LCP measures is accurate
- [ ] Check that the explanation of how Lighthouse gets LCP data is accurate
- [ ] Check if the guidance is relevant. In this case, the guidance is just a link to Phil's "How to improve LCP on your site" section, so it's just a matter of checking if Phil covered all of the important topics.

File: src/site/content/en/lighthouse-performance/lighthouse-largest-contentful-paint/index.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+visible to users. See [Largest Contentful Paint defined][definition] for more
+details on how LCP is determined.
+
+## How Lighthouse determines your LCP score
+
+[Native support for LCP launched in Chrome 77][launch], so Lighthouse doesn't
+need to calculate LCP. It just uses the [Largest Contentful Paint API][api] to get LCP
+data from Chrome.
+
+<!-- TODO explain how the LCP data is mapped to a score between 0 and 100. -->
```
[the remark this answers is not in the document]
Comment (connorjclark) [reply src/site/content/en/lighthouse-performance/lighthouse-largest-contentful-paint/index.md]: All performance scores are green @ 90, yellow @ 50. We haven't determined these curves yet, so I can't yet provide what LCP values these would be.

File: src/site/content/en/lighthouse-performance/lighthouse-largest-contentful-paint/index.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+---
+
+<!-- TODO https://web.dev/largest-contentful-paint already exists.
+     What should the URL for this page be? -->
+
+Largest Contentful Paint (LCP) is one of the metrics
+tracked in the **Performance** section of the Lighthouse report.
+Each metric captures some aspect of page load speed.
+
+<!-- TODO update this depending on how the report displays LCP. Milliseconds? -->
```
[the remark this answers is not in the document]
Comment (connorjclark) [reply src/site/content/en/lighthouse-performance/lighthouse-largest-contentful-paint/index.md]: seconds

File: src/site/content/en/lighthouse-performance/lighthouse-largest-contentful-paint/index.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+<!-- TODO update this once LCP lands in the report UI. -->
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="largest-contentful-paint.png"
+       alt="A screenshot of the Lighthouse Largest Contentful Paint audit">
+</figure>
+
+## What LCP measures
+
+LCP measures when the largest content element visible in the viewport is
```
Comment (connorjclark) [inline src/site/content/en/lighthouse-performance/lighthouse-largest-contentful-paint/index.md]: I don't have any good alternatives, but `largest content element visible in the viewport` is confusing (`visible` makes me need to parse the sentence carefully)

File: src/site/content/en/lighthouse-performance/lighthouse-largest-contentful-paint/index.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+rendered to the screen. This approximates when the main content of the page is
+visible to users. See [Largest Contentful Paint defined][definition] for more
+details on how LCP is determined.
+
+## How Lighthouse determines your LCP score
+
+[Native support for LCP launched in Chrome 77][launch], so Lighthouse doesn't
+need to calculate LCP. It just uses the [Largest Contentful Paint API][api] to get LCP
```
Comment (connorjclark) [inline src/site/content/en/lighthouse-performance/lighthouse-largest-contentful-paint/index.md]: This isn't true - we look at the trace directly. We could have used the API, but we also need to look at the trace for lantern / simulation reasons.

File: src/site/content/en/lighthouse-performance/lighthouse-largest-contentful-paint/index.md (hunk 4 above)
Comment (kaycebasques) [reply src/site/content/en/lighthouse-performance/lighthouse-largest-contentful-paint/index.md]: Thanks, I think I can just remove `visible`

**Comment by kaycebasques** [comment]
Thank you @connorjclark for the review!

**Comment by kaycebasques** [comment]
Blocked until the audit lands in Lighthouse.

**Pull request merged**